DemoApp/Sources/Extensions/CallViewModel+Snapshot.swift

```diff
@@ -20,6 +28,43 @@ extension CallViewModel {
             }
         }
     }
+
+    private func resize(
+        image: UIImage,
+        to targetSize: CGSize
+    ) -> UIImage? {
+        guard
+            image.size.width > targetSize.width || image.size.height > targetSize.height
+        else {
+            return image
+        }
+
+        let widthRatio = targetSize.width / image.size.width
+        let heightRatio = targetSize.height / image.size.height
+
+        // Determine the scale factor that preserves aspect ratio
+        let scaleFactor = min(widthRatio, heightRatio)
+
+        let scaledWidth = image.size.width * scaleFactor
+        let scaledHeight = image.size.height * scaleFactor
+        let targetRect = CGRect(
+            x: (
+                targetSize.width - scaledWidth
+            ) / 2,
```

Comment:
Formatting looks a little bit odd here compared to `y`

Reply:
I'm wondering why swift-format didn't catch it. That happened because i pressed ctrl+M on the line. I'll update it